organization/team.md

```diff
@@ -50,6 +50,12 @@ organizing Google Summer of Code for NumFOCUS.
 | Phil Elson         |                 |                |
 | Thomas Caswell     |                 |                |
 
+#### Pandas
+
+| Name                | Melange Profile | GitHub Profile |
+|---------------------|-----------------|----------------|
+| Jeff Reback         |                 |                |
```

Comment:
Done. @jreback Could you let me know if someone didn't receive the email invitation?